docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -473,6 +473,8 @@ Current set of claims definitions:
 | tcb_date                   | oe_datetime_t      | Remote Only   |The date and time when the evidence's TCB level was certified.       |
 | validity_from              | oe_datetime_t      | Remote Only   |Overall datetime from which the evidence and endorsements are valid. |
 | validity_until             | oe_datetime_t      | Remote Only   |Overall datetime at which the evidence and endorsements expire.      |
+| ue_id                      | uint8_t[16]        | Remote Only   |Universal Entity Identity (qe_id for SGX).                           |
```

Comment:
Per https://datatracker.ietf.org/doc/draft-ietf-rats-eat/ section 3.4, 16 bytes is the wrong length for a UEID since a UEID requires a type byte in front.

Reply:
Updated ueid to be 33 bytes, with first byte as 00 for sgx ueid.

Reply:
Type 0x00 does not appear to be a legal value for the first byte.

Reply:
@dthaler thank you for the catch. I have modified the type to be 0x01 (RAND).